.changesets/add-config-appsignal-rb-file-support.md

```diff
@@ -0,0 +1,19 @@
+---
+bump: minor
+type: add
+---
+
+Add `config/appsignal.rb` config file support. When a `config/appsignal.rb` file is present in the app, the Ruby gem will automatically load it when `Appsignal.start` is called.
+
+The `config/appsignal.rb` config file is a replacement for the `config/appsignal.yml` config file. When both files are present, only the `config/appsignal.rb` config file is loaded.
```

Comment:
Is this correct? In the commit message, you describe scenarios 3 and 4, in which `Appsignal.configure` is called, and that causes `appsignal.yml` to be loaded, but not `appsignal.rb`.

Reply:
Hmm yeah it depends on how the config is loaded.

- Appsignal.configure -> loads appsignal.yml file
- Appsignal.start -> loads appsignal.rb file.

Reply:
```suggestion
The `config/appsignal.rb` config file is a replacement for the `config/appsignal.yml` config file. When both files are present, only the `config/appsignal.rb` config file is loaded when the configuration file is automatically loaded by AppSignal.
```